src/utils/material.js

```diff
@@ -28,6 +30,48 @@ module.exports.updateMap = function (shader, data) {
   }
 };
 
+module.exports.updateDistortionMap = function (longType, shader, data) {
```

Comment:
can we add a docstring to this function just describing what it does, its parameters, the longType/shortType mapping?